### ISSUE TITLE
Super-resolution and Inpainting pre-trained models

### ISSUE THREAD

**Issue by TiagoHRPG** [Algolzw/image-restoration-sde#7]
Congratulations for the amazing work!!

How can I have access to the single image super-resolution and inpainting pre-trained models?

**Comment by Algolzw** [comment]
Hi,

Thanks for your interest in our work! All pre-trained models and results are provided [here](https://drive.google.com/drive/folders/14SvJXvp0HPKFHpGaaHy7h2I2fkRnIXAG?usp=share_link). 
Note that the hyperparameter "max_sigma" for super-resolution and inpainting is set to 30.

Best.

**Comment by Algolzw** [comment]
We also uploaded the inpainting mask [here](https://github.com/Algolzw/image-restoration-sde/tree/main/codes/config/inpainting/data/datasets/gt_keep_masks).

**Comment by TiagoHRPG** [comment]
Thanks a lot!

**Comment by Sttrr** [comment]
> Congratulations for the amazing work!!
> 
> How can I have access to the single image super-resolution and inpainting pre-trained models?

When I use inpainting pretrained model, I meet size mismatching problem as following pictures show. Have you encountered the same problem? How did you solve it?
![image](https://github.com/Algolzw/image-restoration-sde/assets/38433751/aa424ff0-16fc-40ce-8e22-1d4364d5a983)

**Comment by Algolzw** [comment]
Hi, it seems that the parameter `nf` should be 32 to match the size in pre-trained model weights. You can modify it [here](https://github.com/Algolzw/image-restoration-sde/blob/6ff7a86d8591ea50dc7db469703ed29f2ab49360/codes/config/inpainting/options/test/ir-sde.yml#L36).
